src/Symmetric/Openssl.php

```diff
@@ -167,6 +167,14 @@ class Openssl implements SymmetricInterface
      */
     protected $tagSize = 16;
 
+    /**
+     * Supported algorithms
+     *
+     * @internal
+     * @var array
```

Comment:
Can we be more precise with this type? `array<int, string>`? `list<string>`?

Reply:
I followed the initial code which is not precise on this point.

Reply:
Yes, and I'm asking for a refinement here, which seems normal? 🤔

Reply:
It is indeed a `list<string>`.

Reply:
BTW, these adjustments are regular/normal in a PR cycle: you can push fixup commits to the same branch, and this patch will be updated 👍